### PR TITLE
Add publicKeyToDidDoc() method.

### PR DESCRIPTION
**Pull request by dmitrizagidulin**

### CODE REVIEW

**Review by JSAssassin — APPROVED** [review]
LGTM, thanks.

File: lib/DidKeyDriver.js [hunk 1]
```diff
@@ -140,11 +140,29 @@ export class DidKeyDriver {
...
    * @typedef LDKeyPair
-   * @param {LDKeyPair} options.keyPair - A verification key pair to use to
-   *   generate the DID Document.
+   * @param {LDKeyPair|object} options.keyPair - Key used to generate the DID
+   *   document (either an LDKeyPair instance containing public key material,
+   *   or a "key description" plain object (such as that generated from a KMS)).
+   *
+   * @returns {Promise<object>} Resolves with the generated DID Document.
+   */
+  async publicKeyToDidDoc({keyPair} = {}) {
```
Comment (gannan08) [inline lib/DidKeyDriver.js:154]: The function signature invites developers to use poor hygiene when dealing with key material. The function name is `publicKeyToDidDoc` which is fine... but it takes the parameter `keyPair` which suggests that the function requires the public/private key pair. Though the jsdoc nudges the developer to use the public key material, I suggest more explicitness for our Public API.

```suggestion
  async publicKeyToDidDoc({publicKeyDescription} = {}) {
```

**Review by gannan08 — CHANGES_REQUESTED** [review]
The function signature invites developers to use poor hygiene when dealing with key material. The function name is `publicKeyToDidDoc` which is fine... but it takes the parameter `keyPair` which suggests that the function requires the public/private key pair. Though the jsdoc nudges the developer to use the public key material, I suggest more explicitness for our Public API.

```suggestion
  async publicKeyToDidDoc({publicKeyDescription} = {}) {
```

File: lib/DidKeyDriver.js (hunk 1 above)
Comment (dmitrizagidulin) [reply lib/DidKeyDriver.js:154]: @gannan08 - sure thing, happy to change the signature to your suggestion.

Comment (dmitrizagidulin) [reply lib/DidKeyDriver.js]: @gannan08 - renamed.

File: CHANGELOG.md [hunk 2]
```diff
@@ -44,49 +51,30 @@ const keyAgreementKeyPair = methodFor({purpose: 'keyAgreement'});
...
 const keyPair = await Ed25529VerificationKey2020.generate();
-const did = `did:key:${keyPair.fingerprint()}`;
-const didDocument = await didKeyDriver.get({did});
+const {didDocument} = await didKeyDriver.publicKeyToDidDoc({publicKeyDescription: keyPair});
 ```
 
-Don't forget that you can use the `didKeyDriver.publicMethodFor({purpose})` 
-after turning a key to a DID Document with `.get()`:
+Don't forget that you can use the `didKeyDriver.publicMethodFor({purpose})`
+method to fetch a particular key, after creating the did document.
```
Comment (gannan08) [inline CHANGELOG.md:75]:
```suggestion
method to fetch a particular key, after creating the DID Document.
```

**Review by mattcollier — APPROVED** [review]

**Pull request merged**